command/agent/host/windows.go

```diff
@@ -36,34 +37,30 @@ func mountedPaths() (disks []string) {
 }
 
 type df struct {
-	size  int64
-	avail int64
+	size       *uint64 // "systemFree" less quotas
+	avail      *uint64
+	systemFree *uint64
 }
 
 func makeDf(path string) (*df, error) {
-	h, err := syscall.LoadDLL("kernel32.dll")
-	if err != nil {
-		return nil, err
-	}
-
-	c, err := h.FindProc("GetDiskFreeSpaceExW")
-	if err != nil {
-		return nil, err
-	}
-
 	df := &df{}
+	err := windows.GetDiskFreeSpaceEx(
+		syscall.StringToUTF16Ptr(path),
+		df.avail, df.size, df.systemFree)
```

Comment:
If instead of making all of the fields on df pointers, could we pass them by refere here?
```suggestion
		&df.avail, &df.size, &df.systemFree)
```

That would allow skipping the nil checks below, but otherwise I think it's functionally the same. :thinking:

Reply:
Oh, yeah that's nicer. Will fix.

Reply:
Done and double-checked that we're still getting the same results, out of justified paranoia at Windows :grinning: